apps/docs/content/components/OrderedList/index.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Ordered list
+description: Use the ordered list component to render a list of ordered items.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image showing an ordered list with different bullet points.](https://github.com/primer/brand/assets/912236/5cda0c94-c120-4fc6-8306-8b55012dee0a)
+
+## Usage
+
+Use an ordered list to represent a list of items in a numerical order. Each item within the ordered list represents an item that is ordered in a sequence and displays a number to indicate its position in the sequence.
+
+The ordered list is commonly used when presenting sequential or step-by-step information. It is suitable for instructions, procedures, tutorials, or any content that follows a specific order. Ensure that the items in the ordered list are logically organized and follow a clear progression. This helps readers understand the sequence of information and makes it easier to follow.
+
+<DoDontContainer>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/170e2098-4c9a-41ce-b8de-7b1e1403b3a4" />
```

Comment:
missing alt text here?

Reply:
Good point. The caption bellow should work as descriptive text, I'll add the `role="presentation"` to the `<img>`